### ISSUE TITLE
Support parallel running protection mods

### ISSUE THREAD

**Issue by SmallJoker** [SmallJoker/simple_protection#13]
The support for them is very poor right now and should be extended to allow server admins to use multiple protection mods for different area sizes or features.

*This is a note to myself for the future development but PRs are always welcome ;)*

**Comment by BrunoMine** [comment]
Sorry. I am very bad at understanding the English language. Are you referring to the streets?

**Comment by BrunoMine** [comment]
Would you accept it from PR? I would like to help.
But maybe you do not like interference as big as that in your mod.

**Comment by SmallJoker** [comment]
@BrunoMine No, this issue is about using multiple protection mods in one world. For example: simple_protection + areas, simple_protection + protector or even simple_protection + areas + protector (if "areas" and "protector" can be combined too).
However, if this was possible, you could protect the spawn region with one large "areas" area and the players could use the 16x80x16 simple_protection areas to protect their houses.

**Comment by BrunoMine** [comment]
Are you referring to claiming protections within an area that is already protected (by other mods like "areas" and "protector")?

**Comment by SmallJoker** [comment]
@BrunoMine Yes. Also the other way: claiming an area in "areas" which is already claimed.

**Comment by ApolloX939** [comment]
This is a hard thing to handle, possible questions needing an answer too:

- areas/protector don't follow any area claiming rules (unlike us, where we split everything into "chunks" as it were)
- which mod takes precedence over the other mods, or do we try to absolutely co-exist (if we have a claimed area, do we prevent protector nodes and area claiming within our claim?)

> And more as they come up
